Library/Homebrew/rubocops/no_fileutils_rmrf.rb

```diff
@@ -0,0 +1,35 @@
+# typed: true
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Homebrew
+      # This cop checks for the use of `FileUtils.rm_f`, `FileUtils.rm_rf`, or `FileUtils.rmtree_f`
+      # and recommends the non-`f` versions.
+      class NoFileutilsRmrf < Base
+        extend AutoCorrector
+
+        MSG = "Use `FileUtils.rm`, `FileUtils.rm_f` or `FileUtils.rmtree` instead of `FileUtils.rm_rf`, `FileUtils.rm_f`, or `FileUtils.rmtree_f`."
```

Comment:
There is no `rmtree_f` afaik. Regular `rmtree` is however bad.

Reply:
D'oh. I blame the non-existent jet lag.

Reply:
```suggestion
        MSG = "Use `FileUtils.rm`, `FileUtils.rm_f` or `FileUtils.rmtree` instead of `FileUtils.rm_rf`, `FileUtils.rm_f`, or `Pathname#rmtree`."
```

Reply:
Accidentally clicked edit rather than quote but yeah added my suggestion to the above.

Reply:
I see, there are two `rmtree`s. I'll come back to this later!